### ISSUE TITLE
RCA QC check?

### ISSUE THREAD

**Issue by tberardini** [geneontology/go-annotation#1581]
Our docs:

http://www.geneontology.org/page/rca-inferred-reviewed-computational-analysis

Updated November 9, 2007

> Note: Annotations using the RCA code should be reviewed after one year, any older than this date will be deleted.
> 

However, there's no check that implements this 'deletion rule'.

I looked here:

http://geneontology.org/page/annotation-quality-control-checks

Is there a rule in place that's missing from the documentation or is there no QC in place for this?

**Comment by cmungall** [comment]
> http://geneontology.org/page/annotation-quality-control-checks

This should be made to redirect to https://github.com/geneontology/go-site/blob/master/metadata/rules/

Add we need to add a RCA check here

**Comment by tberardini** [comment]
We can still get the check into place in under 10 years after it was last updated. :)

**Comment by tberardini** [comment]
And thanks, @cmungall, for your speedy reply to my query.

**Comment by vanaukenk** [comment]
A number of groups have reviewed their RCA annotations, and while many are removing those annotations, there are some bona fide RCA annotations associated with published papers that are staying.  See #1801 for details.
Since annotation groups cannot realistically replicate published analyses, I suggest that we not implement the one-year filtering rule, but instead strongly encourage groups to, whenever possible, use other sources for annotation and like any of their annotations, review RCA annotations periodically for accuracy.
There are still RCA annotations from groups that do not regularly attend the annotation calls; I will contact these groups directly.

**Comment by rachhuntley** [comment]
If it looks like the RCA code is going to be kept, I have a potential use case for the extracellular matrix annotations from proteomic experiments, which I have detailed in this ticket https://github.com/geneontology/go-annotation/issues/1796.

**Comment by rachhuntley** [comment]
Hi Kimberley @vanaukenk,
If this looks like a reasonable use case, I will go ahead and request a more specific ECO term as mentioned in #1796 
Thanks.

**Comment by vanaukenk** [comment]
Hi @rachhuntley 
Yes, I think this is a reasonable use case.  
Is it correct, though, that this use case incorporates both experimental and computational evidence?  If so, I wonder if we need to do a little work with ECO here to flesh out the combinatorial evidence part of the ontology to be clearer on what different types of evidence are being combined.
We would still need to make sure that the new codes map up to RCA, though, in order to be GAF-compliant....

**Comment by rachhuntley** [comment]
Yes, that's correct. Hopefully we can just make the new term a child of both the experimental and computational codes. I will open the ticket and see if they agree.

**Comment by vanaukenk** [comment]
Okay, thanks @rachhuntley 
Let me know if at any point you think we need to discuss things on an annotation call.

**Comment by ValWood** [comment]
out of date
